examples/federated-learning/tff/nvflare/docker-image/Dockerfile

```diff
@@ -0,0 +1,11 @@
+#checkov:skip=CKV_DOCKER_2: No need for HEALTHCHECK
+FROM nvflare/nvflare:2.4.1
```

Comment:
Add this file to dependabot.yml so we watch it for updates.

Reply:
Done

Reply:
where? :)